cmake/PACE_Docs.cmake

```diff
@@ -10,34 +10,56 @@ set(MANUAL_OUTPUT_DIR "${DOCS_WORK_DIR}/latex" CACHE FILEPATH "Directory to put
 find_program(sphinx-build NAMES sphinx-build)
```

Comment:
There's no file-level docstring documenting which (if any) variables the PACE_Docs function requires to be set and which is sets and which build targets it adds (`docs`, `manual`, `docs-pack`)